src/client/scripts/cardSubmitModal.js

```diff
@@ -62,26 +44,40 @@ Template.cardSubmitModal.events({
     }
 });
 
+function getCardData(eve){
```

Comment:
3 methods again, getThought, getCategory, getAuthor.  Return object arrays like I do not like.  If it is going to be done, return a JSON or something.